dali/server/daldap.cpp

```diff
@@ -144,24 +146,50 @@ class CDaliLdapConnection: implements IDaliLdapConnection, public CInterface
         Owned<ISecUser> user = ldapsecurity->createUser(username);
         user->credentials().setPassword(password);
 
-        //Check user's digital signature, if present
         bool authenticated = false;
-        if (!isEmptyString(udesc->querySignature()))
+
+        //Check that the digital signature provided by the caller (signature of
+        //caller's "username;timeStamp") matches what we expect it to be
+        if (!isEmptyString(udesc->queryUserTimeStampSignature()))
         {
             if (nullptr == pDSM)
                 pDSM = createDigitalSignatureManagerInstanceFromEnv();
             if (pDSM && pDSM->isDigiVerifierConfigured())
             {
-                StringBuffer b64Signature(udesc->querySignature());
-                if (!pDSM->digiVerify(username, b64Signature))//digital signature valid?
+                StringBuffer requestTimestamp;
+                udesc->queryUTCTimeStamp().getString(requestTimestamp, false);//extract timestamp string from Dali request
```

Comment:
for 'old' 3.14 clients, no 'utcTimeStamp' will have been deserialized, so requestTimestamp is always going to be 00:00:00...

Reply:
I see now the this isn't relevant, as won't get to this block if there is no user timestamp signature.